tmp.go

```diff
@@ -0,0 +1,9 @@
+package moira
+
+type tmp struct {
+	field1 bool
+	field2 string
+	field3 bool
+	field4 int
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`field4` is unused (structcheck)